sql/core/src/test/resources/sql-tests/inputs/postgreSQL/boolean.sql

```diff
@@ -22,6 +22,7 @@ SELECT false AS `false`;
 
 SELECT boolean('t') AS true;
 
+-- [SPARK-27931] Trim the string when cast string type to boolean type
```

Comment:
we do trim. Do we need to keep the JIRA ID?

Reply:
This is introduced by revert, delete in next commit.